### ISSUE TITLE
Suggestion for Follows Schema

### ISSUE THREAD

**Issue by ghiliweld** [beakerbrowser/unwalled.garden#17]
Instead of having the follows schema only consist of urls, it could also include a nickname we choose to give the person we're following. If several urls have the same name then this would help the user distinguish between the different people they follow who may have the same preferred name.

#### Defs
- preferred name: the name you give yourself (I'd call myself `"ghiliweld"`)
- nickname: the name someone who follows you might give you to help them identify you (someone might prefer to just call me `"ghili"`)

#### Example
```json
{
  "following": [
    { 
      "url" : "dat://43dfc9f23fdded8cc7c01c71c0702a0529130af0258e7fb30bf5a0a3f73d69b3",
      "nickname": "john"
    }
  ]
}
```

**Comment by RangerMauve** [comment]
Yeah, Scuttlebutt has shown that nicknames for people you follow can be useful. They also support custom profile pictures FWIW

**Comment by pfrazee** [comment]
We used a similar mechanism in [SSB](https://www.scuttlebutt.nz/). It was definitely fun - you would see all the known nicknames on somebody's profile. I'm not sure if I think it has a *ton* of utility but I'm open to discussing it.

It might be smart to turn each URL string into an object now, to make something like this easier to roll out if we decide to. Alternatively, info like this could be stored in other schemas.

**Comment by pzwang** [comment]
Would be fun if people allowed bidirectional push from friends to show nicknames on their own profiles. However, need to make sure that this doesn't end up like LinkedIn badges where I can recommend my friends for being "Excellent at Debugging COBOL". :-)

**Comment by danimesq** [comment]
> We used a similar mechanism in SSB. It was definitely fun - you would see all the known nicknames on somebody's profile. I'm not sure if I think it has a ton of utility but I'm open to discussing it.

Showing all known names on user profile, leds to online harassment.
For example, Justin Bieber haves its profile visited by Selena Gomes, and she sees all its names:

> - "gayber"
> - "justin bamby"
> - "femme"
> - "drunk delicate"
> - "justin timberlake"

Showing all added names on profile incentives bullying, which can led to health issues and even suicide.

Adding nicknames and photo to friends should be user-side, like contacts.

**Comment by ghiliweld** [comment]
@DaniellMesquita that's something I didn't consider. You're right, nicknames should only be user-side not bidirectional.
